app/ai/voice/agents/breeze_buddy/memory/backends/pgvector/backend.py

```
@@ -0,0 +1,109 @@
+            for index, operation in enumerate(operations)
+            if operation.op != "DELETE"
+        ]
+        texts = [operations[index].fact for index in embedded_indexes]
+        embeddings = (
+            await get_embedding_provider(embedding_config).embed(
+                texts, input_type="document"
+            )
+            if texts
+            else []
+        )
+        vectors = dict(zip(embedded_indexes, embeddings))
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)